posts/2018-02-09-serverless-and-graphql-in-love.md

```diff
@@ -0,0 +1,560 @@
+---
+title: Serverless and GraphQL in Love  
```

Comment:
lets revisit this / part 1/2

Reply:
Some ideas, like any of these?

Streamline your life with Serverless and GraphQL
Simplify your infrastructure with Serverless and GraphQL
Serverless + GraphQL = an HTTP endpoint you can rely on
Use Serverless + GraphQL for a secure and scalable endpoint

Reply:
I'll think more, lets circle back on this.